.readthedocs.yaml

```diff
@@ -11,6 +11,10 @@ build:
     post_checkout:
       - git fetch --tags
 
+    # Replace versions in sponsor URLs
+    pre_build:
+      - cog -rP docs/index.md
```

Comment:
@hynek would you be open to sticking `tox` into the RTD config and making it as close as possible to the CI and local runs? I did it in a few places and liked how it simplifies the setup and makes it more predictable.

Reply:
I would, it would just complicate the RTD setup :|

Reply:
fixed!